### PR TITLE
development of bluegenes ESYN wrapper

### PR DESCRIPTION
**Pull request by Sakshisrivastava413**

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "accepts": ["id"],
```
Comment (heralden) [inline config.json]: I think this should be `ids` instead.

**Review by heralden — CHANGES_REQUESTED** [review]
Small change needed to `config.json`; otherwise looks good!

The example data gives `No interactions found. Have you tried expanding your search or using a different data source?` but this might be because Humanmine recently got a new release, changing the object IDs.

**Comment by Sakshisrivastava413** [comment]
> The example data gives `No interactions found. Have you tried expanding your search or using a different data source?` but this might be because Humanmine recently got a new release, changing the object IDs.

Right. That message comes from esyn.org as the passed objectIDs were not found.

**Review by heralden — APPROVED** [review]
Thanks, looks good now. :100:

**Pull request merged**